app/mainscreen/page.tsx

```diff
@@ -0,0 +1,9 @@
+import MonthlyView from "./monthlyview/MonthlyView";
+
+export default function MainScreen() {
+  return (
+    <section className="bg-[#E8E8E8] h-full flex flex-col gap-y-6">
```

Comment:
See my comment above regarding the colors, this one is very close to a color we already have , gray-100.
so you can use bg-gray-100